app/scripts/controllers/courses/CoursesShowCtrl.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+define(['paw'], function(paw) {
+  paw.controller('CoursesShowCtrl', ['$routeParams', function($routeParams) {
+    var _this = this;
+    var courseId = $routeParams.courseId; // For future Service calls
+
+    this.formatDate = function(finalExamDate) {
+      var date = finalExamDate.split('T')[0];
+      var time = finalExamDate.split('T')[1];
+      return date.split('-')[2] + '/' + date.split('-')[1] + '/' + date.split('-')[0] + ' ' + time;
+    };
+
+    this.course = {
+      'courseId': '93.42',
+      'credits': 6,
+      'name': 'Física II',
+      'semester': 3
+    };
+
+    this.correlatives = [
```

Comment:
Correlatives information should be related to the course, because they are correlatives of that course. Nest this correlatives inside the above course definition. That's the way data should be received by the controller when fetching it from the Courses service.